samples/deployment/vmware/README.md

```diff
@@ -0,0 +1,66 @@
+# Setup Guide
+
+1. Create virtual env
+    ```bash
+    virtualenv -p python3.7 ./venv
+    source venv/bin/activate
+    pip install ansible pyvmomi
+    ```
+
+1. Install vsphere automation python sdk 
+    ```bash
+    git clone https://github.com/vmware/vsphere-automation-sdk-python.git
+    cd vsphere-automation-sdk-python
+    pip install --upgrade --force-reinstall -r requirements.txt --extra-index-url file://`pwd`/lib
+    ```
+1. Set correct values for following demo_cloud.vmware.yaml:
+    ```yaml
+    plugin: "vmware_vm_inventory"
+    hostname: "vmware.api.host.com"
+    username: "username"
+    password: "passwd"
+    ```
+
+1. Verify results
+    ```bash
+    ansible-inventory -i demo_cloud.vmware.yaml --list
+    ```
+
+# Prepare playbook
```

Comment:
Usually, the document has only one H1 section with its name at the very beginning of the file. Other subsections are either H2/H3/etc.

Reply:
fixed